.jenkins/pytorch/win-test-helpers/build_pytorch.bat

```diff
@@ -37,33 +37,19 @@ if "%VC_VERSION%" == "" (
 @echo on
 popd
 
-if "%CUDA_VERSION%" == "9" goto cuda_build_9
-if "%CUDA_VERSION%" == "10" goto cuda_build_10
-if "%CUDA_VERSION%" == "11" goto cuda_build_11
-goto cuda_build_end
+if not "%USE_CUDA%"=="1" goto cuda_build_end
 
-:cuda_build_9
+set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v%CUDA_COMPLETE_VERSION%
 
-set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.2
-set CUDA_PATH_V9_2=%CUDA_PATH%
+rem version transformer, for example 10.1 to 10_1. But I don't find where the CUDA_PATH_VXX_X is used 
+set VERSION_SUFFIX=%CUDA_COMPLETE_VERSION:.=_%
+set CUDA_PATH_V%VERSION_SUFFIX%=CUDA_PATH
 
-goto cuda_build_common
-
-:cuda_build_10
-
-set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.1
-set CUDA_PATH_V10_1=%CUDA_PATH%
-
-goto cuda_build_common
-
-:cuda_build_11
-
-set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.1
-set CUDA_PATH_V11_1=%CUDA_PATH%
-
-goto cuda_build_common
-
-:cuda_build_common
+set CUDNN_LIB_DIR=%CUDA_PATH%\lib\x64
+set CUDA_TOOLKIT_ROOT_DIR=%CUDA_PATH%
+set CUDNN_ROOT_DIR=%CUDA_PATH%
+set NVTOOLSEXT_PATH=C:\Program Files\NVIDIA Corporation\NvToolsExt
```

Comment:
Some of the lines have repeated more than once. Could you please do a cleanup?

Reply:
do you mean it's duplicated of https://github.com/pytorch/pytorch/blob/5b6b1495b9bc27ddb2cf10df6a387090716eb16a/.jenkins/pytorch/win-test-helpers/setup_pytorch_env.bat#L77-L84

Reply:
Yeah.

Reply:
Repeated variable definitions:
NVTOOLSEXT_PATH: Line 51, 57
CUDA_TOOLKIT_ROOT_DIR: Line 49, 55
CUDNN_ROOT_DIR: Line 50, 56
CUDNN_LIB_DIR: Line 48, 54